Lib/idlelib/configdialog.py

```diff
@@ -672,18 +674,18 @@ def create_page_general(self):
             frame
                 frame_run: LabelFrame
                     startup_title: Label
-                    (*)radio_startup_edit: Radiobutton - startup_edit
-                    (*)radio_startup_shell: Radiobutton - startup_edit
+                    (*)startup_editor_on: Radiobutton - startup_edit
+                    (*)startup_shell_on: Radiobutton - startup_edit
```

Comment:
If we were starting fresh..., but we are not.  We cannot change config-main.def.
[General]
editor-on-startup= 0
This is read as a boolean, and startup_editor should be BooleanVar, not IntVar. 

What we show users  When tests are merged, we can review code and layout.  I intend to remove three top frame and change to:

Window to open at startup:                        O Shell                        O Editor
Initial size:                                                   Width [______]              Height [_____]
When run code in editor: ... (see bpo-19042 for possible change)